<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogWatcherClass</class>
 <widget class="QMainWindow" name="LogWatcherClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogWatcher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" rowspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QRadioButton" name="radioButtonNext">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>NEXT</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButtonIntellect">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>INTELLECT</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="openButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>OPEN</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonOpenSearch">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>SEARCH</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="checkBoxDebug">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>DEBUG</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxDebug">
              <item>
               <property name="text">
                <string>White</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Red</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Yellow</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Cyan</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Violet</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Purple</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Magenta</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Lavender</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Gray</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Green</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Blue</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="checkBoxInfo">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>INFO</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxInfo">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <item>
               <property name="text">
                <string>White</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Red</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Yellow</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Cyan</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Violet</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Purple</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Magenta</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Lavender</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Gray</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Green</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Blue</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="3">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QCheckBox" name="checkBoxError">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>ERROR</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxError">
              <item>
               <property name="text">
                <string>White</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Red</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Yellow</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Cyan</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Violet</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Purple</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Magenta</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Lavender</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Gray</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Green</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Blue</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QCheckBox" name="checkBoxWarning">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>WARNING</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxWarning">
              <item>
               <property name="text">
                <string>White</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Red</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Yellow</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Cyan</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Violet</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Purple</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Magenta</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Lavender</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Gray</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Green</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Blue</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="4">
           <widget class="QPushButton" name="pushButtonAnalyz">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>ANALYZE</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButtonEditData">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>EDIT DATABASE</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>145</width>
     <height>0</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LogWatcher.qrc"/>
 </resources>
 <connections/>
</ui>
